README.md

```diff
@@ -166,8 +166,21 @@ plan_hierarchy:
     data_hash: yaml_data
 ```
 
+
 See the following [documentation](https://puppet.com/docs/bolt/latest/hiera.html#outside-apply-blocks) for further explanation.
 
+#### Using indictor_exclusion with a code manager and hiera.yaml
```

Comment:
Using Static Hiera data to populate indictor_exclusions when executing the plans

Reply:
Replacing the title now